src/client/pps/pps.proto

```diff
@@ -308,6 +312,7 @@ service API {
   rpc InspectJob(InspectJobRequest) returns (JobInfo) {}
   rpc ListJob(ListJobRequest) returns (JobInfos) {}
   rpc DeleteJob(DeleteJobRequest) returns (google.protobuf.Empty) {}
+  rpc StopJob(StopJobRequest) returns (google.protobuf.Empty) {}
```

Comment:
Should we maybe call this `CancelJob` or `FailJob`, to me `StopJob` makes it sound like the job could be restarted, which is how it works with `StopPipeline`